Drop-In/src/main/java/com/braintreepayments/api/dropin/AddCardActivity.java

```diff
@@ -374,9 +374,8 @@ public void onError(Exception error) {
             } else if (error instanceof DownForMaintenanceException) {
                 mBraintreeFragment.sendAnalyticsEvent("sdk.exit.server-unavailable");
             }
-
-            finish(error);
         }
+        finish(error);
```

Comment:
We wouldn't want to finish this activity on any error. Field errors are processed on L361 for example which causes a state change in this activity.

Reply:
I see now.  This change would effectively change the activity making the feedback on the field annoyingly un actionable.  Let me take another wack at this then.